### PR TITLE
- updated RHEL/CentOS installation docs

### PR DESCRIPTION
**Pull request by paszczus**

### CODE REVIEW

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 1]
```diff
@@ -20,15 +20,15 @@ service mysqld start
 **MariaDB**
 ```bash
 yum install net-snmp mariadb-server mariadb-client
-chkconfig mariadb on
-service mariadb start
+systemctl enable mariadb
+systemctl start mariadb
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: These two wont work for CentOS 6 MariaDB. Retain `chkconfig` and `service` they will work with CentOS 7 as well afaik.

**Comment by paszczus** [comment]
Indeed. I will fix this, thanks for pointing that out!

**Comment by paszczus** [comment]
@f0o now it should be OK

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 2]
```diff
@@ -193,23 +202,41 @@ To prepare the web interface (and adding devices shortly), you'll need to create
...
-
-    chmod 775 rrd
+```
 
 > If you're planing on running rrdcached, make sure that the path is also chmod'ed to 775 and chown'ed to librenms:librenms.
 
+**SELinux**
+> if you're using SELinux you need to allow nginx to write into logs directory.
+> semanage tool is a part of policycoreutils-python, so if don't have it, you can install it
+
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: Please note here that SELinux is still experimental to us, we cant guarantee it working yet

**Comment by MindTooth** [comment]
I'll do a clean install based on the new steps on Monday, if you would like a second input?

**Comment by paszczus** [comment]
@f0o i have added a note about SELinux, however i`m testing it on my production with SELinux so if everything will be working fine i will remove that note, what do you think? But for now i think this pull can be merged

**Comment by laf** [comment]
@paszczus 

Where you have:

`service mariadb start (systemctl start mariadb on CentOS 7)`

Doesn't service mariadb start still work in CentOS 7? I thought it did?

If not and we need to keep that can you split out to:

CentOS 5/6:
`service mariadb start`

CentOS 7:
`systemctl start mariadb`

**Comment by paszczus** [comment]
IT will work but this is not proper way to do this, systemctl should be used.
Spliting out that will also require to split it in other parts (nginx, apache etcetera)...

**Comment by laf** [comment]
Then I think we need to either stick with just system X start or split it out, those commands are designed to be copied and pasted and people will blindly just copy and paste them.

**Comment by laf** [comment]
Sorry (didn't mean to close)

**Comment by paszczus** [comment]
@laf ok, please check out now. I have separated centos 6 and 7 commands

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 3]
```diff
@@ -20,15 +21,34 @@ service mysqld start
 **MariaDB**
 ```bash
 yum install net-snmp mariadb-server mariadb-client
-chkconfig mariadb on
-service mariadb start
+chkconfig mariadb on (systemctl enable mariadb on CentOS 7)
+service mariadb start (systemctl start mariadb on CentOS 7)
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: These two became redundant with the ones further down. Remove?

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 4]
```diff
@@ -69,22 +89,43 @@ Install necessary software.  The packages listed below are an all-inclusive list
 
 Note if not using HTTPd (Apache): RHEL requires `httpd` to be installed regardless of of `nginx`'s (or any other web-server's) presence.
 
+**CentOS 6**
+```bash
     yum install epel-release
-    yum install php php-cli php-gd php-mysql php-snmp php-pear php-curl httpd net-snmp graphviz graphviz-php mysql ImageMagick jwhois nmap mtr rrdtool MySQL-python net-snmp-utils vixie-cron php-mcrypt fping git
+yum install php php-cli php-gd php-mysql php-snmp php-pear php-curl httpd net-snmp graphviz graphviz-php mysql ImageMagick jwhois nmap mtr rrdtool MySQL-python net-snmp-utils vixie-cron php-mcrypt fping git
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: Please re-add the indent

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 5]
```diff
@@ -151,15 +203,19 @@ server {
...
     cd /opt/librenms
+```
 
 At this stage you can either launch the web installer by going to http://IP/install.php, follow the on-screen instructions then skip to the 'Web Interface' section further down. Alternatively if you want
 to continue the setup manually then just keep following these instructions.
 
+```bash
     cp config.php.default config.php
-    vim config.php
+    vim config.
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: filename should be `config.php` :)

File: doc/Installation/Installation-(RHEL-CentOS).md (hunk 3 above)
Comment (paszczus) [inline doc/Installation/Installation-(RHEL-CentOS).md]: sure, i`m missed that. thanks

File: doc/Installation/Installation-(RHEL-CentOS).md (hunk 4 above)
Comment (paszczus) [inline doc/Installation/Installation-(RHEL-CentOS).md]: done

File: doc/Installation/Installation-(RHEL-CentOS).md [hunk 6]
```diff
@@ -193,24 +249,53 @@ To prepare the web interface (and adding devices shortly), you'll need to create
...
     # For Nginx:
     chown nginx:nginx logs
-
-    chmod 775 rrd
+```
 
 > If you're planing on running rrdcached, make sure that the path is also chmod'ed to 775 and chown'ed to librenms:librenms.
 
+**SELinux**
+> if you're using SELinux you need to allow nginx to write into logs directory.
```
Comment (f0o) [inline doc/Installation/Installation-(RHEL-CentOS).md]: It's not only `nginx` that needs to be able to write into logs :)

Could we make it more bold that we dont currently support SELinux at all and that we're still figuring a correct policy?

**Comment by f0o** [comment]
Wrote some last notes, happy to merge after those are sorted/discussed :+1:

**Comment by paszczus** [comment]
fixes made

**Pull request merged**